modules/statics/README.md

```diff
@@ -15,15 +15,15 @@
 
 ### Get the number of decimal places in a Bitcoin
 
-#### Javascript
+####javascript
```

Comment:
These look like broken section headers to me now when I preview this file.

Reply:
![image](https://user-images.githubusercontent.com/23717670/143297202-bbc4d9bd-e463-42b2-a86d-d854f4e723b5.png)

Reply:
Whoopsie

My brain's markdown parser malfunctioned, I thought that was a code block. Un momento

Reply:
Fixed 3 occurrences